site/src/pages/components/inputnumber.research.mdx

```diff
@@ -0,0 +1,22 @@
+---
+menu: Research
+name: Number Input
```

Comment:
We're historically leaving out the word `input` so let's shorten this to just Number for now.